core/src/Streamly/Internal/Data/MutArray/Type.hs

```diff
@@ -2678,19 +2678,19 @@ data SpliceState s arr
 -- immutable array never has additional space so a new array is allocated
 -- instead of mutating it.
 
--- | Scan @rCompactChunksLE n@ coalesces adjacent arrays in the input stream
+-- | Scan @rCompactLE n@ coalesces adjacent arrays in the input stream
 -- only if the combined size would be less than or equal to n.
 --
 -- /Internal/
-{-# INLINE_NORMAL rCompactChunksLE #-}
-rCompactChunksLE :: (MonadIO m, Unbox a)
+{-# INLINE_NORMAL rCompactLE #-}
+rCompactLE :: (MonadIO m, Unbox a)
```

Comment:
See previous comments on naming and comments on the other related PR regarding naming.